grpc-server/src/main/scala/com/avast/sst/grpc/server/GrpcServerModule.scala

```diff
@@ -0,0 +1,44 @@
+package com.avast.sst.grpc.server
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.{Resource, Sync}
+import io.grpc.{Server, ServerBuilder, ServerInterceptor, ServerServiceDefinition}
+
+import scala.collection.immutable.Seq
+import scala.concurrent.ExecutionContext
+object GrpcServerModule {
+
+  /** Makes [[io.grpc.Server]] (Netty) initialized with the given config, services and interceptors.
+    *
+    * @param services service implementations to be added to the handler registry
+    * @param executionContext executor to be used for the server
+    * @param globalInterceptors that are run for all the services
+    */
+  @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+  def make[F[_]: Sync](config: GrpcServerConfig,
+                       services: Seq[ServerServiceDefinition],
+                       executionContext: ExecutionContext,
+                       globalInterceptors: Seq[ServerInterceptor] = List.empty): Resource[F, Server] =
```

Comment:
Maybe the `global` prefix is not required.

Reply:
I wanted to express that it's applied to all the services but you are right that it's not very nice.